cpp/src/slice2cpp/Gen.cpp

```diff
@@ -940,12 +934,16 @@ Slice::Gen::validateMetadata(const UnitPtr& u)
     };
     knownMetadata.emplace("cpp:dll-export", std::move(dllExportInfo));
 
-    // "cpp:doxygen:include"
+    // "cpp:doxygen"
+    // The metadata validation system does not support metadata names with colons. So here, for
+    // cpp:doxygen:include:Ice/Ice.h, include is either the first of two args, or part of the one arg. We consider it's
+    // a single arg that starts with 'include:'.
     MetadataInfo doxygenInfo = {
         .validOn = {typeid(Unit)},
-        .acceptedArgumentKind = MetadataArgumentKind::RequiredTextArgument,
+        .acceptedArgumentKind = MetadataArgumentKind::SingleArgument,
```

Comment:
commas are allowed in include statements.

Reply:
What do you mean?

Reply:
Sorry, I meant: commas are _not_ allowed in include statements. And whether or not to accept commas is the only difference between RequiredTextArgument (allows commas) and SingleArgument (does not allow commas).